tests/unit/inference/test_work_counting.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from transformers import AutoConfig
+
+from zeroband.inference.work_counting import get_inference_input_output_flops_deepseek_v3, get_inference_input_output_flops_qwen3
+
+
+@pytest.mark.parametrize(
+    "model_name_or_path, active_params",
+    [
+        ("deepseek-ai/DeepSeek-R1-0528", 37e9),  # 37B Active Params from https://arxiv.org/pdf/2412.19437
+    ],
+)
+def test_get_inference_input_output_flops_deepseek_v3(model_name_or_path: str, active_params: int):
```

Comment:
Is it a test than need to download v3 ? If yes need to mark it "very slow" and probably skip it from ci